ee/vellum_ee/workflows/display/nodes/get_node_display_class.py

```diff
@@ -11,9 +12,17 @@ def get_node_display_class(
         node_display_class = base_class.get_from_node_display_registry(node_class)
     except KeyError:
         try:
-            return get_node_display_class(
+            base_node_display_class = get_node_display_class(
                 base_class, node_class.__bases__[0], node_class if root_node_class is None else root_node_class
             )
+
+            # `base_node_display_class` is always a Generic class, so it's safe to index into it
+            NodeDisplayBaseClass = base_node_display_class[node_class]  # type: ignore[index]
+            NodeDisplayClass = types.new_class(
+                f"{node_class.__name__}Display",
+                bases=(NodeDisplayBaseClass,),
+            )
+            return NodeDisplayClass
```

Comment:
For any node classes where the user did not specify a display class, we dynamically generate one that subclasses our out of box (or previously registered) display classes. This maintains the invariant we said above

Reply:
Neat